src/applications/ChildBenefitsClaim/StartPage.tsx

```diff
@@ -1,19 +1,21 @@
-import React from 'react'
-import Button from '../../components/BaseComponents/Button/Button'
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Button from '../../components/BaseComponents/Button/Button';
 
 const StartPage: React.FC<{onStart: React.MouseEventHandler, onBack: any}> = ({onStart, onBack}) => {
+  const { t } = useTranslation();
```

Comment:
Is there any particular reason for using {t} as constant name, if not please change it.

Reply:
t is short for translate